### ISSUE TITLE
Clarify Features on Homepage

### ISSUE THREAD

**Issue by deeplow** [freedomofpress/dangerzone.rocks#19]
From user testing we realized that all the features need some rewording.

![Screenshot 2023-05-24 at 08-37-06 IJF Dangerzone user testing (4_2023)](https://github.com/freedomofpress/dangerzone.rocks/assets/47065258/69248a9e-e097-4f63-baa1-e8e7a8a760ed)

**No network access** - one participant in a user study was confused about the meaning of sandbox. The original text assumes some prior knowledge of what a sandbox is. And two users mentioned that they thought the tool was "uploading" the document We can try to frame it as opening documents on your computer but in a more secure compartment isolated from the internet.

**Optional OCR** - this one has two potential issues: (1) people may not know what OCR is, to begin with. We should focus instead of saying that the document will be searchable, (which is what one pretends). Furthermore, it may be confusing for some why this is even needed — if the document was originally searchable.

**Reduced File Size** - Similarly to the "optional OCR", this is written of someone who already knows Qubes Trusted PDF and its limitations, but for someone coming fresh into Dangerzone, the files at the end may look much larger than the original. This can lead to confusion and should be clarified.

**Open Docs Safely** - we mention here that people can make Dangerzone their default document viewer, but currently, Dangerzone may [not be very usable for previewing files](https://github.com/freedomofpress/dangerzone/issues/424) as one has to go through the conversion process first.

**Comment by apyrgio** [comment]
I totally agree with the breakdown of what is unclear in the above features. While we can better explain the above features, maybe we need a different list, that will be more tailored to potential Dangerzone users (journalists, whistleblowers, privacy-minded individuals).

I would recommend the following features, in order of importance:

* **Malware / Tracker removal:** Dangerzone converts files in a process that's akin to taking a picture of their pages and presenting them as a PDF. The resulting PDF is stripped from any kind of malware and trackers (e.g., canary tokens).
* **Source protection:** Dangerzone recreates the original document and does not retain any file metadata that may point to the original source (e.g., dates, names, GPS coordinates)
* **Many supported document and image formats:** Dangerzone can convert the following file types: PDF, doc, ...
* **Privacy:** The document conversion takes place in a local, offline sandbox, that runs within your computer. Your document will not be shared with any third-party, and the conversion can take place in air-gaped computers as well.

**Comment by deeplow** [comment]
> I totally agree with the breakdown of what is unclear in the above features. While we can better explain the above features, maybe we need a different list, that will be more tailored to potential Dangerzone users (journalists, whistleblowers, privacy-minded individuals).

Yes! That's pretty much what I was thinking. And I think your suggested features are spot on.
